agentops/llms/llama_stack_client.py

```diff
@@ -0,0 +1,235 @@
+import inspect
+import pprint
+import sys
+from typing import Dict, Optional
+
+from ..event import LLMEvent, ErrorEvent, ToolEvent
+from ..session import Session
+from ..log_config import logger
+from agentops.helpers import get_ISO_time, check_call_stack_for_agent_id
+from .instrumented_provider import InstrumentedProvider
+
+class LlamaStackClientProvider(InstrumentedProvider):
+    original_complete = None
+    original_create_turn = None
+    
+
+    def __init__(self, client):
+        super().__init__(client)
+        self._provider_name = "LlamaStack"
+
+    def handle_response(self, response, kwargs, init_timestamp, session: Optional[Session] = None, metadata: Optional[Dict] = {}) -> dict:
```

Comment:
Is kwargs passed as dictionary here?
with kwargs I intend keyword arguments, so seeing a packed `kwargs` confuses me because it is commonly used in decorator patterns not elsewhere